crates/ethereum-rpc/src/lib.rs

```diff
@@ -51,503 +53,613 @@ pub struct SyncStatus {
-    //                 m.gas_price = gas_price.unwrap();
-
-    //                 TypedTransactionRequest::Legacy(m)
-    //             }
-    //             Some(TypedTransactionRequest::EIP2930(mut m)) => {
-    //                 m.chain_id = chain_id;
-    //                 m.gas_limit = gas_limit;
-    //                 m.gas_price = gas_price.unwrap();
-
-    //                 TypedTransactionRequest::EIP2930(m)
-    //             }
-    //             Some(TypedTransactionRequest::EIP1559(mut m)) => {
-    //                 m.chain_id = chain_id;
-    //                 m.gas_limit = gas_limit;
-    //                 m.max_fee_per_gas = max_fee_per_gas.unwrap();
-
-    //                 TypedTransactionRequest::EIP1559(m)
-    //             }
-    //             Some(TypedTransactionRequest::EIP4844(mut m)) => {
-    //                 m.chain_id = chain_id;
-    //                 m.gas_limit = gas_limit;
-    //                 m.max_fee_per_gas = max_fee_per_gas.unwrap();
-
-    //                 TypedTransactionRequest::EIP4844(m)
-    //             }
-    //             None => return Err(EthApiError::ConflictingFeeFieldsInRequest.into()),
-    //         };
-
-    //         // get raw transaction
-    //         let transaction = to_primitive_transaction(transaction_request)
-    //             .ok_or(SignError::InvalidTransactionRequest)?;
-
-    //         // sign transaction
-    //         let signed_tx = ethereum
-    //             .eth_signer
-    //             .sign_transaction(transaction, from)
-    //             .map_err(|e| to_jsonrpsee_error_object(e, ETH_RPC_ERROR))?;
-
-    //         RlpEvmTransaction {
-    //             rlp: signed_tx.envelope_encoded().to_vec(),
-    //         }
-    //     };
-    //     let (tx_hash, raw_message) = ethereum
-    //         .make_raw_tx(raw_evm_tx)
-    //         .map_err(|e| to_jsonrpsee_error_object(e, ETH_RPC_ERROR))?;
-
-    //     ethereum.add_messages(vec![raw_message]);
-
-    //     Ok::<_, ErrorObjectOwned>(tx_hash)
-    // })?;
-
-    rpc.register_blocking_method::<Result<Vec<GethTrace>, ErrorObjectOwned>, _>(
-        "debug_traceBlockByHash",
-        move |parameters, ethereum, _| {
-            let mut params = parameters.sequence();
-
-            let block_hash: B256 = params.next()?;
-            let evm = Evm::<C>::default();
-            let mut working_set = WorkingSet::new(ethereum.storage.clone());
-            let opts: Option<GethDebugTracingOptions> = params.optional_next()?;
-
-            let block_number =
-                match evm.get_block_number_by_block_hash(block_hash, &mut working_set) {
-                    Some(block_number) => block_number,
-                    None => {
-                        return Err(EthApiError::HeaderNotFound(block_hash.into()).into());
-                    }
-                };
+    fn eth_accounts(&self) -> RpcResult<Vec<Address>> {
+        Ok(self.ethereum.eth_signer.signers())
+    }
```

Comment:
remove this function please

Reply:
remove. remove the local flag also. we don't have those features right now. and looks like we won't need to